lib/snail.rb

```diff
@@ -22,10 +22,12 @@ class UnknownAttribute < ArgumentError; end
   attr_accessor :region
   attr_accessor :postal_code
   attr_reader   :country
+  attr_reader   :country_iso
 
   # Store country as ISO-3166 Alpha 2
   def country=(val)
-    @country = Snail.lookup_country_iso(val)
+    @country = val
+    @country_iso = Snail.lookup_country_iso(val)
```

Comment:
i'd like to make this change more backwards compatible. let's go with `@country = Snail.lookup_country_iso(val) || val` and drop the `@country_iso` ivar. this should mean that behavior will only change for unknown countries: anyone with a standardized data will be unaffected, and anyone with mixed data will hopefully see an improvement in formatting.